docs/source/public/overview.rst

```diff
@@ -499,14 +496,92 @@ This script will:
+* Gather additional simulation information
+   * Additional simulation metrics, including transformation and ligand scores.
 
 Industry partners will be expected to submit this information back to OpenFE for
 analysis. Please note that all data will be collected in a human readable format
 in order to allow industry partners the ability to review the data ahead of submission
 back to the OpenFE team.
 
+For performing the data gathering, please note the following:
+
+* Make sure to run the `post-simulation cleanup`_ script before running the data gathering script! If you have not run the cleanup script, this data gathering script will throw an error.
+* The data gathering script will throw an error if the network is broken and also not all three repeats have finished successfully.
+
+   * If you have a broken network, please run the `fix_networks`_ script and the new edges first to obtain a connected network.
+   * If some of your repeats had failed, please re-run those as described `here <failed_edges>`_
+
+
+.. code-block:: bash
+
+   wget https://raw.githubusercontent.com/OpenFreeEnergy/IndustryBenchmarks2024/main/industry_benchmarks/utils/data_gathering.py
+   python data_gathering.py --input_alchemical_network network_setup/alchemical_network.json --results-folder results_0 results_1 results_2
+
+If the `fix network script <fix_networks>`_ was run, two alchemical networks need to be provided to the data gathering script, the original one and the one that was created after running the ``fix_networks.py``:
+
+.. code-block:: bash
+
+   python data_gathering.py --input_alchemical_network network_setup/alchemicalNetwork/alchemical_network.json --fixed_alchemical_network new_network_setup/alchemicalNetwork/alchemical_network.json --results-folder results_0 results_1 results_2
+
+If confidential ligand names were used, those can be replaced with generic ligand names. To do that you will need to add the flag ``--hide-ligand-names``:
+
+.. code-block:: bash
+
+   python data_gathering.py --input_alchemical_network network_setup/alchemicalNetwork/alchemical_network.json --results-folder results_0 results_1 results_2 --hide-ligand-names
+
+.. warning::
+   Per default, the ligand names that were present in the original ligand sdf will be stored. Please check those and if they contain any confidential information, use the ``hide-ligand-names`` flag.
+
+.. _gathered_data:
+
+Content of the data that will be transferred to OpenFE
+------------------------------------------------------
+
+Here you can find the full list of files and data that will be extracted in the `data gathering script <gathering_of_results>`_.
+
+1. A file containing a variety of network properties (``all_network_properties.json``). More specifically, the file contains the following information:
+
+  * ``Network_map``: A network containing the ligand names as nodes, and the transformations that were run between those ligands.
```

Comment:
```suggestion
  * ``Network_map``: A network containing the ligand names as nodes, and ligand name pairs as edges to specify which transformations were run.
```

Reply:
I would consider a slight rewording to make it sound less like we know how to transform between the ligands.